### ISSUE TITLE
Remove on MwBuffer doesn't seem to work

### ISSUE THREAD

**Issue by akxcv** [openplanet-nl/issues#58]
Hey! I was playing around with automating replay rendering, and wanted to use `app.Editor.PluginAPI` which is `CGameEditorMediaTrackerPluginAPI@` when you are editing a replay. 

Sadly, it looks like not a single method of this object is doing anything.

I am trying to remove the "Custom camera" track from the replay. I've also tried using `Remove(idx)` on `PluginAPI.Clip.Tracks` `MwBuffer`, but that also does not do anything in game. I don't see other alternatives.

I'm not sure if it's a bug or just the way Trackmania works.

**Comment by codecat** [comment]
I don't think this is a bug, if those functions don't do anything then Nadeo's code isn't doing anything. After all, the functions just call the game's code.

Calling `Remove()` on `MwBuffer` should work though, perhaps there is something wrong there.

**Comment by akxcv** [comment]
After further experimentation:

`Remove` and `Remove_Range` do work on `<CGameCtnMediaTrack>.Blocks`, and the changes do reflect in game (although that's not helpful to my purposes)

`Remove` and `Remove_Range` do not appear to work on `PluginAPI.Clip.Tracks`. When I looped through the buffer in code after using `Remove`, it was indeed missing the elements I'd removed, the issue is it's not reflected in game.

**Comment by codecat** [comment]
Well, `CGameCtnMediaClip::Clips` is `const`, so it's normal that you wouldn't be able to remove or add anything to it.

Although, so is `CGameCtnMediaTrack::Blocks`.. I don't really see any obvious difference between those, so perhaps it's just the game that doesn't accept stuff being removed from there.
